include/contractor/graph_contractor_adaptors.hpp

```diff
@@ -128,23 +128,25 @@ ContractorGraph toContractorGraph(NodeID number_of_nodes, InputEdgeContainer inp
 
 template <class Edge, typename GraphT> inline std::vector<Edge> toEdges(GraphT graph)
 {
-    std::vector<Edge> edges;
-    edges.reserve(graph.GetNumberOfEdges());
-
-    util::UnbufferedLog log;
-    log << "Getting edges of minimized graph ";
-    util::Percent p(log, graph.GetNumberOfNodes());
-    const NodeID number_of_nodes = graph.GetNumberOfNodes();
-    if (graph.GetNumberOfNodes())
+    util::Log() << "Converting contracted graph with " << graph.GetNumberOfEdges()
+                << " to edge list (" << (graph.GetNumberOfEdges() * sizeof(Edge) / 1024. / 1024.)
```

Comment:
imho, it is better to show the value in bytes to avoid MiB vs MB confusion
```
" edges to an edges list (allocating "  << graph.GetNumberOfEdges() * sizeof(Edge) << " bytes)";
```